packages/bierzo-wallet/src/communication/extensionRpcEndpoint.ts

```diff
@@ -70,6 +75,11 @@ export const extensionRpcEndpoint: RpcEndpoint = {
 
     return new Promise((resolve, reject) => {
       chrome.runtime.sendMessage(config.extensionId, request, response => {
+        if (chrome.runtime.lastError) {
+          resolve(undefined);
+          return;
+        }
+
         if (response === undefined) {
```

Comment:
Please put both blocks under common condition. `if (chrome.runtime.lastError || response === undefined) { ...`

Reply:
I tested that, I put it separately because legibility because:

- Removes the error:
`if (chrome.runtime.lastError || response === undefined) { ...`
- Does not remove the error:
`if (response === undefined || chrome.runtime.lastError) { ...`

But if it's more idiomatic that way all change it no problem.

Reply:
Looking at the [documentation of chrome.runtime.sendMessage](https://developer.chrome.com/apps/runtime#method-sendMessage):

> The JSON response object sent by the handler of the message. If an error occurs while connecting to the extension, the callback will be called with no arguments and runtime.lastError will be set to the error message. 

you see that `response` is either a JSON object (i.e. not `undefined`) or an error occurred. This means we can as well remove the `response === undefined`, since it happens exactly when `chrome.runtime.lastError` is set.